assets/src/application/ui/layout/entityList/pagination/useEntityPagination.ts

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import { slice } from 'ramda';
+
+const useEntityPagination = ({ entities }) => {
+	const [perPage, setPerPage] = useState(6);
+	const initialPaginatedEntities = slice(0, perPage)(entities);
+	const [paginatedEntities, setPaginatedEntities] = useState(initialPaginatedEntities);
+	const total = entities.length;
+	// const pageNumber = parseInt(currentPage, 10);
+	// const lastPage = Math.ceil(entities.length / perPage);
+	// const actualPageNumber = pageNumber <= lastPage ? pageNumber : lastPage;
+
+	// useEffect(() => {
+	// 	if (pageNumber > 1 && pageNumber > lastPage) {
+	// 		setCurrentPage(lastPage);
+	// 	}
+	// }, [perPage, currentPage, entities.length]);
```

Comment:
let's just remove all of these comments since they are not needed moving forwards... if someone wants to refer back to older code, then they can look in the git log

Reply:
cleaned up, thx